daemon/stop.go

```diff
@@ -36,7 +36,13 @@ func (daemon *Daemon) ContainerStop(ctx context.Context, name string, options co
 }
 
 // containerStop sends a stop signal, waits, sends a kill signal.
-func (daemon *Daemon) containerStop(ctx context.Context, ctr *container.Container, options containertypes.StopOptions) (retErr error) {
+func (daemon *Daemon) containerStop(_ context.Context, ctr *container.Container, options containertypes.StopOptions) (retErr error) {
```

Comment:
Been going back-and-forth whether to remove the context.Context here, but decided to go for the most minimal changes to fix the regression to reduce code-churn (and we may want to have the context available here in future for other purposes).